### DISCUSSION TITLE
yank the file path on Windows

### DISCUSSION THREAD

**Discussion by azinsharaf** [gokcehan/lf#1320] [Q&A] [answered]
how can i copy the full file path into the clipboard on Windows os?

**Comment by joelim-work** [comment]
The simple answer is pipe the value of `%f%` into `clip`:

```
map Y &echo %f% | clip
```

However this leaves the path wrapped in quotes (as paths can contain spaces), and also appends a new line. This can be solved by using `%f:"=%` to substitute quotes with blanks, and `echo | set /p=` to prevent a new line from being appended:

```
map Y &echo | set /p=%f:"=%| clip
```

---

For PowerShell users, `Set-Clipboard` can be used:

```powershell
map Y &Set-Clipboard -Value ($Env:f -replace '"')
```

**Reply by azinsharaf** [reply]
do you know how the command would be on windows?
the first one is : `map Y &echo $Env:f | clip`

**Reply by joelim-work** [reply, marked as answer]
The config snippets I provided above were for Windows CMD. From the looks of it you're using PowerShell, in which case you can simply use:

```powershell
map Y &Set-Clipboard -Value ($Env:f -replace '"')
```
